airbyte-integrations/connectors/source-marketo/acceptance-test-config.yml

```diff
@@ -0,0 +1,27 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-marketo:dev
+tests:
+  spec:
+    - spec_path: "source_marketo/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+#  basic_read:
```

Comment:
nit: comments are needed, why this tests are skipped.

Reply:
Updated.